tensorflow_addons/metrics/hamming.py

```diff
@@ -38,14 +38,14 @@ def hamming_distance(actuals: TensorLike, predictions: TensorLike) -> tf.Tensor:
 
     Usage:
 
-    ```python
-    actuals = tf.constant([1, 1, 0, 0, 1, 0, 1, 0, 0, 1],
-                          dtype=tf.int32)
-    predictions = tf.constant([1, 0, 0, 0, 1, 0, 0, 1, 0, 1],
-                              dtype=tf.int32)
-    result = hamming_distance(actuals, predictions)
-    print('Hamming distance: ', result.numpy())
-    ```
+    >>> actuals = tf.constant([1, 1, 0, 0, 1, 0, 1, 0, 0, 1],
+    ... dtype=tf.int32)
+    >>> predictions = tf.constant([1, 0, 0, 0, 1, 0, 0, 1, 0, 1],
+    ... dtype=tf.int32)
+    >>> metric = hamming_distance(actuals, predictions)
+    >>> print('Hamming distance: ', metric.numpy())
```

Comment:
ditto

Reply:
Metric does not have any attribute result. SO, I only change the print statement.

> Thanks for the contribution! Some requests:
> 
> 1. Can we canonicalize naming convention? Like
> 
> ```python
> >>> y_true = np.foo(xxx)
> >>> y_pred = np.foo(xxx)
> >>> metric = tfa.metrics.bar(yyy)
> >>> metric.update_state(y_true, y_pred)
> >>> result = metric.result()
> >>> result.numpy()
> ```
> 
> 1. Some comments are not so meaningful in testable docstring. Like `# metric name`: When users see the API page, they know this page is that metric. Therefore, we can also clean up this kind of comments in docstring in this PR!

To the best of my knowledge, I have not encountered `# metric name`: in this Pr